app/views/case_contacts/_case_contact.html.erb

```diff
@@ -42,6 +42,21 @@
       <%= contact.creator&.display_name %>
     <% end %>
   </td>
+  <td class="read-more">
+    <span class="mobile-label">Notes</span>
+    <%- if contact.notes && contact.notes.length > 100 %>
+      <div class="limited-notes" style="display: block;">
+        <%= contact.decorate.notes %>
+        <span><a href='#' onClick='moreText(event)'>Read more</a></span>
```

Comment:
For this `Read more` toggle, I used a link tag with an `onClick` handler. A link doesn't seem semantically appropriate given this is more action-based, but stylistically it seems what we would want. 
Perhaps I should add a `role: 'button'`?

Reply:
onClick seems reasonable
+1 to `role: button` :)